### PR TITLE
[chore](cloud) add selected partition ids and visible version in log

### PR DESCRIPTION
**Pull request by sollhui**
## Proposed changes

Add selected partition ids and visible version in log.

### CODE REVIEW

**Comment by sollhui** [comment]
run buildall

**Comment by sollhui** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/planner/OlapScanNode.java [hunk 1]
```diff
@@ -731,6 +731,11 @@ private Collection<Long> distributionPrune(
 
     // Update the visible version of the scan range locations.
     public void updateScanRangeVersions(Map<Long, Long> visibleVersionMap) {
+        if (ConnectContext.get() != null) {
+            LOG.info("query id: {}, selectedPartitionIds: {}, visibleVersionMap: {}",
```
Comment (liaoxin01) [inline fe/fe-core/src/main/java/org/apache/doris/planner/OlapScanNode.java:735]: Many logs may be printed, using debug level maybe better.

**Comment by sollhui** [comment]
run buildall

**Comment by sollhui** [comment]
run buildall

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**